05/DB.test.js

```diff
@@ -1 +1,130 @@
-import DB from './DB';
\ No newline at end of file
+import DB from './DB';
+
+describe('DB', () => {
+	describe('insert', () => {
+		it('should reject when id is not a number', async () => {
+			expect.assertions(1);
+			const db = new DB();
+
+			try {
+				await db.insert({ id: 'a' });
+			} catch (err) {
+				expect(err).toBe('ID can be only number!');
+			}
+		});
+
+		it('should reject when given id exist in DB', async () => {
+			expect.assertions(1);
+			const db = new DB();
+			await db.insert({ id: 1 });
+
+			try {
+				await db.insert({ id: 1 });
+			} catch (err) {
+				expect(err).toBe("ID can't be duplicated!");
+			}
+		});
+
+		it('should return 1 when insert data to empty array', async () => {
+			const db = new DB();
+			const data = { a: 1, b: 2 };
+			await db.insert(data);
+
+			expect(db._rows.length).toEqual(1);
```

Comment:
Lepiej wykorzystać getRows() bo `_rows` (podkreślnik) oznacza, żeby nie wykorzystywać tego elementu - mówi się, że to "pole prywatne".